e2e/framework/test_context.go

```diff
@@ -246,6 +246,12 @@ func (r ServiceLogin) IsEmpty() bool {
 	return r.Username == "" && r.Password == ""
 }
 
+// SiteAddress
```

Comment:
incomplete comment.
Also, `Site` is the old vocabulary - this should be called `ClusterAddress`